cluster/store/store.go

```diff
@@ -433,7 +438,10 @@ func (st *Store) WaitForAppliedIndex(ctx context.Context, period time.Duration,
 			if idx = st.lastAppliedIndex.Load(); idx >= version {
 				return nil
 			} else {
-				st.log.Debug("wait for update version", "got", idx, "want", version) // TODO-RAFT remove
+				st.log.WithFields(logrus.Fields{
+					"got":  idx,
+					"want": version,
+				}).Debug("wait for update version") // TODO-RAFT remove
```

Comment:
**suggestion (non-blocking):** let's remove the `// TODO-RAFT remove` given that we are using that log line just for debugging purposes